src/index.html

```diff
@@ -1,13 +1,240 @@
+            town.
+          </p>
+          <p class="article--price">$ 2 590</p>
+        </article>
+        <article class="article">
+          <img src="./images/photos/bike3.jpg"
+               class="pictures"
+               alt="bike sporty 4" />
+          <h3 class="article--name">Agile ride 3</h3>
+          <p class="article--description"> The lightweight frame that has earned its street tread as a sleek, agile
+            ride.
+          </p>
+          <p class="article--price">$ 2 090</p>
+        </article>
+      </section>
+
+      <section class="section">
+        <h2 class="title title--section">The details</h2>
+        <article class="article">
+          <div class="pictures">
+            <img src="./images/photos/accessories1.jpg"
+                 class="pictures__item"
+                 alt="mobile phone with active appsense" />
+            <img src="./images/photos/accessories2.jpg"
+                 class="pictures__item"
+                 alt="holder for attaching to the handlebar" />
+          </div>
+          <h3 class="article--name">Auto Unlock</h3>
+          <p class="article--description"> The app senses when you're nearby to unlock automatically. GPS tracking so
+            you know where your bike is and can track it anytime.
+          </p>
+        </article>
+        <article class="article">
+          <div class="pictures">
+            <img src="./images/photos/accessories3.jpg"
+                 class="pictures__item"
+                 alt="mobile phone with active appsense" />
+            <img src="./images/photos/accessories4.jpg"
+                 class="pictures__item"
+                 alt="holder for attaching to the handlebar" />
+          </div>
+          <h3 class="article--name">Range & Integrated lights</h3>
+          <p class="article--description"> The removable battery has up to 70km battery autonomy and weighs only 2.4 kg.
+            Lights integrated into the frame give you always-on visibility day and night.
+          </p>
+        </article>
+        <article class="article">
+          <div class="pictures">
+            <img src="./images/photos/accessories5.jpg"
+                 class="pictures__item"
+                 alt="mobile phone with active appsense" />
+            <img src="./images/photos/accessories6.jpg"
+                 class="pictures__item"
+                 alt="holder for attaching to the handlebar" />
+          </div>
+          <h3 class="article--name">Hydraulic disc brakes & Lightweight</h3>
+          <p class="article--description"> Brakes with total stopping power the second you make contact. The removable
+            battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you
+            always-on visibility day and night.
+          </p>
+        </article>
+        <a href="#"
+           class="button">Explore</a>
```

Comment:
fix formatting
add 40px space before button as it is in figma

Reply:
I will formatting after finish mobile all components